generators/app/templates/scripts/_postcss.config.js

```diff
@@ -15,13 +18,26 @@ const printOutput = postcss.plugin('postcss-print-output', function(opts) {
     const relativeFrom = path.relative(process.cwd(), results.opts.from);
     const relativeTo = path.relative(process.cwd(), results.opts.to);
 
-    console.log(`${relativeFrom} -> ${relativeTo}`);
+    console.log(`${relativeFrom} -> ${relativeTo} in ${getNow() - startTime}ms`);
+  };
+});
+
+/**
+ * A function to set the startTime of postcss so that
+ * we can print the time taken in the output.
+ */
+const setTime = postcss.plugin('postcss-set-time', function(opts) {
```

Comment:
looks like there's a plugin that will do some of this already? https://github.com/postcss/postcss-devtools

Reply:
will test

Reply:
This plugin makes the build much slower and also prints out debug information that would probably only be useful when actually developing postcss.

Reply:
cool. Maybe at some point we can make that a more fully featured postcss plugin like time-grunt?